init.sqf

```diff
@@ -218,3 +211,20 @@ f_var_viewDistance_crewOnly = true;
 // [[UnitName1, "UnitClass1"]] spawn f_fnc_disableThermals;
 
 // ====================================================================================
+
+if (isServer) then {

```

Comment:
I feel like this should maybe be built into e.g. famInit rather than being a big blob of code at the bottom of init.sqf. If we want this to be optional (MMs can turn off the swap behaviour), I'd instead suggest a boolean parameter to famInit that an MM could flip instead